x-pack/test/observability_ai_assistant_api_integration/tests/knowledge_base/knowledge_base_user_instructions.spec.ts

```diff
@@ -91,59 +92,64 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       });
 
       it('"editor" can retrieve their own private instructions and the public instruction', async () => {
-        const res = await observabilityAIAssistantAPIClient.editorUser({
-          endpoint: 'GET /internal/observability_ai_assistant/kb/user_instructions',
+        await retry.try(async () => {
+          const res = await observabilityAIAssistantAPIClient.editorUser({
+            endpoint: 'GET /internal/observability_ai_assistant/kb/user_instructions',
+          });
+
+          const instructions = res.body.userInstructions;
+
+          const sortByDocId = (data: any) => sortBy(data, 'doc_id');
```

Comment:
can we avoid `any` here?

Reply:
ah I noticed this is not your change, but imho we can very easily not use `any` here, so hopefully that is a small change

Reply:
Sure @dgieselaar 
I updated it.